src/medication-dispense/medication-dispense.resource.tsx

```diff
@@ -117,6 +142,12 @@ export function initiateMedicationDispenseBody(
         ],
         substitution: {
           wasSubstituted: false,
+          reason: [
+            {
+              coding: [{ code: null }],
+            },
+          ],
+          type: { coding: [{ code: null }] },
```

Comment:
We will probably need to tweak this based on:

https://talk.openmrs.org/t/setting-property-values-to-null-when-submitting-fhir-objects-via-fhir2-module/38471/3